pkg/jobstore/boltdb/store.go

```diff
@@ -325,7 +335,55 @@ func (b *BoltJobStore) getJobs(tx *bolt.Tx, query jobstore.JobQuery) ([]models.J
 		return nil, err
 	}
 
-	return b.getJobsWithinLimit(result, query), nil
+	// Sort the jobs according to the query.SortBy and query.SortOrder
+	listSorter := func(i, j int) bool {
+		switch query.SortBy {
+		case "id":
+			if query.SortReverse {
+				// what does it mean to sort by ID? (rj) it's mostly meaningless and just
+				// gives us a lexicographic sort of the UUIDs.
+				return result[i].ID > result[j].ID
+			} else {
+				return result[i].ID < result[j].ID
+			}
```

Comment:
I can't think of good reasons for users to sort by `id`, specially that it is auto-generated by us. Does it make sense to remove this to simplify our indexing story and pagination story? We might introduce it later when we allow users to provide their `id` where it can mean something